styles/dataDashboard.css

```diff
@@ -88,6 +88,7 @@ body {
 
 .row {
     margin-top: .5em;
+    display: block;
```

Comment:
Still small viewport so we want the items within the row element to stack instead of sit next to each other with flex.